### PR TITLE
Added README file

### PR DESCRIPTION
**Pull request by HovorukhaBohdan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,56 @@
...
++ _`POST: /api/orders` - endpoint for placing orders (User authority)_
++ _`PATCH: /api/orders/{id}` - endpoint for updating orders status (Admin authority)_
++ _`GET: /api/orders/{orderId}/items` - endpoint for getting order items from specific order (User authority)_
++ _`GET: /api/orders/{orderId}/items/{itemId}` - endpoint for getting specific item from certain order (User authority)_
++ _`GET: /api/cart` - endpoint for getting all items in your shopping cart (User authority)_
++ _`POST: /api/cart` - endpoint for adding items in your shopping cart (User authority)_
++ _`PUT: /api/cart/cart-items/{itemId}` - endpoint for updating items quantity (User authority)_
++ _`DELETE: /api/cart/cart-items/{itemId}` - endpoint for deleting items from your shopping cart (User authority)_
+___
+## 🚀 ___Getting Started___ 🚀
```
Comment (d1sam) [inline README.md:49]: what about docker-compose? )

File: README.md [hunk 2]
```diff
@@ -1 +1,56 @@
...
+___
+## 👨‍💻 ___TECHNOLOGIES___ 👨‍💻
++ ***Spring boot***
++ ***Spring Data JPA***
++ ***Spring MVC***
++ ***Spring Security***
++ ***Hibernate***
++ ***MYSQL DB***
++ ***Test containers***
++ ***Maven***
```
Comment (d1sam) [inline README.md]: please provide categories, that technologies related to.

File: README.md
Comment (d1sam) [inline README.md:1]: Don't forget to mention any challenges you faced and how you overcame them.

Comment (d1sam) [reply README.md:1]: please also add Possible improvements section.

**Review by d1sam — CHANGES_REQUESTED** [review]
Good job, but there are some improvements should be done!

File: README.md [hunk 3]
```diff
@@ -11,46 +11,74 @@ ___
...
 + _`PUT: /api/cart/cart-items/{itemId}` - endpoint for updating items quantity (User authority)_
 + _`DELETE: /api/cart/cart-items/{itemId}` - endpoint for deleting items from your shopping cart (User authority)_
 ___
 ## 🚀 ___Getting Started___ 🚀
 ___
-1. **Grab a copy: Start by making a local copy of the project's repository.**
-2. **Database setup: Customize the database settings to match your environment in the application.properties file.**
-3. **Table creation: Make sure the necessary database tables are automatically created using Liquibase.**
-4. **Building and running: Bring the project to life using your favorite IDE or by running the command `mvn spring-boot:run`**
-5. **Exploring the API: Access the API documentation through Swagger, available at http://localhost:8080/swagger-ui.html: http://localhost:8080/swagger-ui.html.**
```
Comment (yevheniisynytsia) [inline README.md]: Let's leave the swagger link
it is useful for reviewing your project

**Review by yevheniisynytsia — APPROVED** [review]
Well done! I left a minor comment, please address it

**Pull request merged**